app/move/validators.test.js

```diff
@@ -0,0 +1,48 @@
+const validators = require('./validators')
+
+describe('Validators', function() {
+  describe('#time()', function() {
+    describe('invalid values', function() {
+      const inputs = [
+        10,
+        null,
+        '10',
+        '25:00',
+        '10:65',
+        'asdf',
+        '10:00a',
+        '9:00p',
+        '9pm',
+        '7am',
+        'midnight',
+        'midday',
+      ]
+
+      inputs.forEach(i => {
+        it(`test for: "${i}"`, function() {
+          expect(validators.time(i)).not.to.be.ok
+        })
+      })
+    })
+
+    describe('valid values', function() {
+      const inputs = [
+        '',
+        '9:00am',
+        '9:00pm',
+        '9:00',
```

Comment:
Aren't 9:00 and 10:00 just as ambiguous as 9 and 10?

Reply:
In what sense? As a valid time? `9:00` is at least formatted more like a time than just an integer.